aws/logs_monitoring/lambda_function.py

```diff
@@ -255,7 +268,7 @@ def compileRegex(rule, pattern):
 DD_CUSTOM_TAGS = "ddtags"
 DD_SERVICE = "service"
 DD_HOST = "host"
-DD_FORWARDER_VERSION = "2.3.4"
+DD_FORWARDER_VERSION = get_env_var("DD_FORWARDER_VERSION", "unknown")
```

Comment:
Why using unknown here? If the forwarder version is not set, we could still default on something known aka 3.0.0 here.

Reply:
I was merely trying to avoid setting it in two places -- the template and source code, which could potentially result in unmatched versions set. Especially given that this fallback logic rarely kicks in, developers could easily forget to update the version number in source code (here).

The reason I extracted version to the template and set it through an environment variable was to make it available to the existing lambda metadata crawler, so we can track the usage internally and present users a nudge to upgrade based on the version. Given this plan, I thought "unknown" is a better fallback, as it also gives us indicator that the forwarder did install through the official approach. 

Let me know what do you think? I can definitely change this back to the version number and document one extra step in our release process to update version in both places. Don't think this is a big deal.